tests/x11/pidgin/prep_pidgin.pm

```diff
@@ -30,7 +30,7 @@ sub run {
     # pidgin main window is hidden in tray at first run
     # need to show up the main window (12-SP2 and SP3)
     # the main window is shown correctly in SLE15
-    if (is_sle('>=15')) {
+    if (is_sle('>=15') or is_tumbleweed) {
         wait_still_screen;
     }
     elsif (is_sle('>=12-sp2')) {
```

Comment:
You can probably drop the condition as well.

Reply:
Like this?
```
if (is_sle('>=12-sp2')) {
        release_key "ctrl-alt";
        assert_and_click "status-icons-pidgin";
}
else {
        wait_still_screen;
}
```

Reply:
Sorry, I probably attached the comment to wrong line/ I had in mind changing `elsif (is_sle('>=12-sp2')) {` into ordinary `else {` as this code should now always run on at least 12-sp4 (which is already reflected in some of your changes).